packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -210,26 +202,19 @@ export class SlackService {
      *      3) Send a duplicate notification to the Nango Admins
      *      4) Add an activity log entry for the notification to the admin account
      */
-    async reportFailure(
-        nangoConnection: NangoConnection,
-        syncName: string,
-        syncType: SyncType,
-        originalActivityLogId: number,
-        environment_id: number,
-        provider: string
-    ) {
+    async reportFailure(nangoConnection: NangoConnection, name: string, type: string, originalActivityLogId: number, environment_id: number, provider: string) {
```

Comment:
I would just question the rename from syncName to name. I'm all for short name when in context but here it's not explicit when used and not explicit when reading this function code.

What motivated this?

Reply:
I changed the name because it isn't always a `syncName` anymore but can be a connection name or a sync name with the addition of `auth` slack notifications